src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -5874,6 +5906,63 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
                         "_lpython_str_startswith", loc);
             }
             return;
+        } else if (attr_name == "endswith") {
+            /* 
+                str.endswith(suffix)     ---->  
+                Return True if the string ends with the specified suffix, otherwise return False.
+            */
+
+            if (args.size() != 1) {
+                throw SemanticError("str.endswith() takes one arguments", loc);
+            }
+
+            ASR::expr_t *arg_suffix = args[0].m_value;
+            ASR::ttype_t *arg_suffix_type = ASRUtils::expr_type(arg_suffix);
+            if (!ASRUtils::is_character(*arg_suffix_type)) {
+                throw SemanticError("str.endswith() takes one arguments of type: str", arg_suffix->base.loc);
+            }
+            
+            if (ASRUtils::expr_value(arg_suffix) != nullptr) {
+                /*
+                    Invoked when Suffix argument is provided as a constant string
+                */
+                ASR::StringConstant_t* suffix_constant = ASR::down_cast<ASR::StringConstant_t>(arg_suffix);
+                std::string suffix = suffix_constant->m_s;
+
+                std :: cout << suffix << " " << s_var << std :: endl;
```

Comment:
This is used for debugging purposes, right?
We need to remove this, I think.

Reply:
Ah, good catch @Thirumalai-Shaktivel . @harshsingh-24 This is the probable cause of C failure. Kindly remove these debugging statements.

Reply:
Ahh, my bad. Must have left it hanging for some last time testing.

Reply:
Yes, it was `cout` that was causing the problems. Now, the builds are successful.